snippets/common/VS_Snippets_CFX/c_identity/common/app1.config

```diff
@@ -1,5 +1,5 @@
 <wsdl:service xmlns:wsdl="blahdeblah"  >
```

Comment:
should we also get rid of the blahdeblah?

Reply:
lol